tools/RestfulApiWebService/DB/Dao/XrefSpecDBContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿using DB.Models;
+using System.Data.Entity;
+
+namespace DB.Dao
```

Comment:
database access obejct

---
In reply to: [131036696](https://github.com/dotnet/docfx/pull/1906#discussion_r131036696) [](ancestors = 131036696)